### PR TITLE
Pull Request Template Updates

### PR DESCRIPTION
**Pull request by garethellis0**
## Please fill out the following before requesting review on this PR

**Description of Change:**
Updating the pull request template

### Checklist
*(Please check every item to indicate your code complies with it (by changing `[ ]`->`[x]`). This will hopefully save both you and the reviewer(s) a lot of time!)*
- [x] **Start of document comments**: each `.cpp` and `.h` file should have a comment at the start of it. See files in `src/sample_package` for examples.
- [x] **Function comments**: All function definitions (usually in the `.h` file) should have a javadoc style comment at the start of them. For examples, see the classes defined in `src/sample_package`
- [x] **Remove all commented out code**
- [x] **Remove extra print statements**: for example, those just used for testing
- [x] **Resolve all TODO's**: All `TODO` (or similar) statements should either be completed or associated with a github issue

***Feel free to make additions of things that we should be checking to this file if you think there's something missing!!!!***

### CODE REVIEW

**Comment by garethellis0** [comment]
Whoops, fixed! Can you re-review?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -4,14 +4,20 @@ This file outlines a list of common things that should be addressed when opening
...
-### Checklist
+### Resolved Issues
+*Link any issues that this PR resolved. Eg `resolves #1, #2, and #5` (note that they MUST be specified like this so Github can automatically close them then this PR merges)*
+
+### Review Checklist
 *(Please check every item to indicate your code complies with it (by changing `[ ]`->`[x]`). This will hopefully save both you and the reviewer(s) a lot of time!)*
-- [ ] **Start of document comments**: each `.cpp` and `.h` file should have a comment at the start of it. See files in `src/sample_package` for examples.
-- [ ] **Function comments**: All function definitions (usually in the `.h` file) should have a javadoc style comment at the start of them. For examples, see the classes defined in `src/sample_package`
+***It is the reviewers responsibility to also make sure every item here has been covered***
+- [ ] **Start of document comments**: each `.cpp` and `.h` file should have a comment at the start of it. See files in the `thunderbots/software/geom` folder for examples.
```
Comment (WilliamGu98) [inline .github/PULL_REQUEST_TEMPLATE.md]: Did you mean to refer to an example from the thunderbots repo?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -4,14 +4,20 @@ This file outlines a list of common things that should be addressed when opening
...
+- [ ] **Function comments**: All function definitions (usually in the `.h` file) should have a javadoc style comment at the start of them. For examples, see the classes defined in `thunderbots/software/geom`
```
Comment (WilliamGu98) [inline .github/PULL_REQUEST_TEMPLATE.md]: Same thing here, not sure if you meant to refer to the thunderbots repo.

Comment (garethellis0) [reply .github/PULL_REQUEST_TEMPLATE.md]: Whoops, good catch! Fixed to point to our examples.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (garethellis0) [reply .github/PULL_REQUEST_TEMPLATE.md]: Fixed to point to our examples.

**Comment by garethellis0** [comment]
Updated and ready for re-review!

**Pull request merged**